src/main/webapp/js/email-browser.js

```diff
@@ -19,6 +19,74 @@
 var jasig = jasig || {};
 
 (function($, fluid) {
+
+	var ischecked = 0;
```

Comment:
is `isChecked` counting the number of checks?
To me `is<something>` expressions are usually boolean.